magenta_docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Uses one of AI Platform base images.
+# You can try using different images however only this one has been tested.
+FROM gcr.io/deeplearning-platform-release/tf2-gpu.2-2
+WORKDIR /root
```

Comment:
Move this close where it's needed.

Reply:
Sorry, I'm not sure if I understand exactly what you mean. Should I move workdir definition to the place where I copy the script?

Reply:
I mean the WORKDIR is used in the unzip below to create temporary files, you can move it before that.

Reply:
Sure, will do

Reply:
Done